mkdocs.yml

```diff
@@ -57,6 +57,7 @@ pages:
     - Errors: api/ErrorHandling.md
 - For Developers:
     - How to Build: Building.md
+    - How to Release: Releasing.md
```

Comment:
These instructions are really for Red (or maybe Greater Batch) use only. Do they really belong among these general developer docs?

Reply:
I concur. Will try to get consensus on a better spot.

Happy to put them wherever folks will actually find them. I did the release from asking around plus memory. The docs I did find after the release were 1) out of date and 2) still weren't being followed. Ex: When I looked through Homebrew we hadn't done a release in a while.